src/components/draw/feature-edit-style-symbole.vue

```diff
@@ -25,21 +22,34 @@ const featureColor = 'red' // feature.olFeature.get('color') // TODO: to plug wh
     </label>
     <!-- Symbole button -->
     <button
-      class="lux-btn flex gap-2 items-center"
-      @click="() => (currentEditCompKey = 'FeatureEditSymbol')"
+      class="lux-btn items-center"
+      @click="() => (popupOpen = true)"
       data-cy="featStyleSymbolEdit"
     >
-      {{ t('Symbole') }}
-      <Circle v-if="featureShape === 'circle'" :fillColor="featureColor" />
-      <Rectangle
-        v-else-if="featureShape === 'square'"
-        :fillColor="featureColor"
-      />
-      <Cross v-else-if="featureShape === 'cross'" :fillColor="featureColor" />
-      <Triangle
-        v-else-if="featureShape === 'triangle'"
-        :fillColor="featureColor"
+      <img
+        class="symbol-style"
+        v-if="!!feature.featureStyle.symbolId"
+        :src="`${SYMBOL_ICONS_URL}/symbol/${feature.featureStyle.symbolId}`"
```

Comment:
Why "symbol/" is not included in the `SYMBOL_ICONS_URL` const?

Reply:
because symbol.url provided by the backend includes symbol/
https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/GSLUX-707_drawStyle/src/components/draw/feature-edit-symbol-list.vue#L55

https://map.geoportail.lu/mymaps/symbols?symboltype=public